libctru/include/3ds/applets/miiselector.h

```diff
@@ -0,0 +1,111 @@
+/**
+ * @file miiselector.h
+ * @brief Mii Selector Applet (appletEd).
+ */
+
+#pragma once
+
+/// @cond INTERNAL
+#define _assert_struct_size(s, size) \
+	_Static_assert(sizeof(s) == (size), #s " is not of size " #size "!")
```

Comment:
@mtheall: Sorry, forgot that this was C11 only. What should this macro expand to if neither of your suggested conditions are met, `((void) 0)`?

@fincs: Of all the exisiting files I would guess `types.h` to be the most fitting, I don't really like here either (just didn't want to touch other files rn). Are there any other places in ctrulib that could benefit from using this macro or similar ones?